uktrade/data-hub-frontend
# Add filter for sub-sector to companies page

## Description of change

Add sub-sector filter to companies page and interactions page

## Test instructions

On the companies page or interactions page, under filters there should be sub-sectors filter. You can search for sub-sectors and filter by them. It should work regardless of what the sectors filter is set to. For example, sector filtered to Aerospace and sub-sector filtered to Music will show companies/interactions that are aerospace or music

## Screenshots
![Screenshot 2023-09-01 at 14 57 51](https://github.com/uktrade/data-hub-frontend/assets/54268863/83f5f534-e464-4286-85a5-de5b3b3b678d)

![Screenshot 2023-09-01 at 15 03 53](https://github.com/uktrade/data-hub-frontend/assets/54268863/3b78088c-28cc-4ac8-9c58-e4fbce87c1ad)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
